spec/features/card_management_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+describe 'Management cards' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.